guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -1,9 +1,9 @@
-[id="Upgrading-Project-from-Enterprise-Linux-7-to-Enterprise-Linux-8_{context}"]
+[id="upgrading-project-in-place-using-leapp_{context}"]
 ifdef::foreman-el,katello[]
-= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+= Upgrading {Project} to Enterprise Linux 8 In-Place Using Leapp
```

Comment:
```suggestion
= Upgrading {Project} to Enterprise Linux 8 in-place using Leapp
```

feels weird to capitalize every word, or is it just me? (same below, obv)

Reply:
It's you, you don't know enough about capitalization in English. Leave this to writers experts, please :D

Reply:
k

Reply:
In headings only prepositions and connectives are lower case.  I know it from song names :-)